src/fragments/lib/push-notifications/react-native/identify_user/20_send_to_pinpoint.mdx

```diff
@@ -1,9 +1,43 @@
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import {
+  identifyUser,
+  IdentifyUserInput
+} from 'aws-amplify/push-notifications';
+
+const userInfo: IdentifyUserInput = {
+  userId,
+  userProfile: {
+    name: 'user',
+    customProperties: {
+      hobbies: ['cooking', 'knitting']
+    }
+  }
+};
+
+await identifyUser(userInfo);
+```
+
+</Block>
+<Block name="JavaScript">
+
 ```js
+import { identifyUser } from 'aws-amplify/push-notifications';
+
 const userInfo = {
-  attributes: {
-    hobbies: ['cooking', 'knitting']
+  userId,
+  userProfile: {
+    name: 'user',
+    customProperties: {
+      hobbies: ['cooking', 'knitting']
+    }
   }
 };
 
-await Notifications.Push.identifyUser(userId, userInfo);
+await identifyUser(userInfo);
```

Comment:
Do we want to show how they can use `options` as well here? Specifically the `userAttributes`?

Reply:
Let's just keep it simple and aligned with how it was previously for now?

Reply:
Think showing the `userAttributes` will be useful here since the previous `attributes` maps to `options->userAttributes` and `customProperties` is a new field that maps to Endpoint's `Attributes` in the service call.